asyncapi/github/service_types.bal

```diff
@@ -49,23 +50,25 @@ public type PullRequestService service object {
     remote function onEdited(PullRequestEvent payload) returns error?;
 };
 
-# Actions related to pull request reviews. The action can be one of submitted, edited or dismissed.
+# Triggers when Choreo recieved a a new event from GitHub related to a GitHub pull request review.
+# Available actions: onSubmitted, onEdited, and onDismissed
 public type PullRequestReviewService service object {
     remote function onSubmitted(PullRequestReviewEvent payload) returns error?;
     remote function onEdited(PullRequestReviewEvent payload) returns error?;
     remote function onDismissed(PullRequestReviewEvent payload) returns error?;
 };
 
-# Actions related to pull request review comments in the pull request's unified diff. 
-# The action can be one of created, edited or deleted. 
+# Triggers when a new event is received from GitHub when a comment is added to a 
+# pull request during a review in the pull request's unified diff view.
+# Available actions: onCreated, onEdited, and onDeleted
 public type PullRequestReviewCommentService service object {
     remote function onCreated(PullRequestReviewCommentEvent payload) returns error?;
     remote function onEdited(PullRequestReviewCommentEvent payload) returns error?;
     remote function onDeleted(PullRequestReviewCommentEvent payload) returns error?;
 };
 
-# Actions related to a release. The action can be one of published, unpublished, 
-# created, edited, deleted, pre-released or released.
+# Triggers when a new event related to a GitHub release is received.
```

Comment:
I think it would more clear if we use ` .. to GitHub release event received` wdyt?